app/templates/views/terms-of-use.html

```diff
@@ -42,7 +42,7 @@ <h2 class="heading-medium" id="when-using-notify">When using Notify</h2>
   </ul>
 
   <h2 class="heading-medium" id="leaving-notify">Leaving Notify</h2>
-  <p class="govuk-body">You can leave Notify at any time. Just <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.support')}}">contact us</a> and we’ll close your account.</p>
-  <p class="govuk-body">When you leave Notify, all your data will be deleted.</p>
+  <p class="govuk-body">You can <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.support')}}">contact us</a> at any time to close your account.</p>
+  <p class="govuk-body">Find out <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.privacy#how-long-we-keep-your-data')}}">how long we keep your personal data</a>.</p>
```

Comment:
```suggestion
  <p class="govuk-body">Find out <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.privacy')}}#how-long-we-keep-your-data">how long we keep your personal data</a>.</p>
```

Just a small code change here - we need the anchor outside of the `{{ }}` part for it to work correctly :)

Reply:
Thanks Sam. That's exactly the help I was asking for in this morning's stand up. :-)